src/tests/deepEqualInAnyOrder.js

```diff
@@ -271,4 +271,22 @@ describe('equalInAnyOrder', () => {
     };
     expectToNotDeepEqualInAnyOrder(a, b);
   });
+
+  it('prepends message from expect', () => {
+    expect(
+      () => expect(true, 'message').to.deep.equalInAnyOrder(false),
+    ).to.throw().and.satisfy(e => /^message:/.test(e.message));
+  });
+
+  it('prepends message from equalInAnyOrder', () => {
+    expect(
+      () => expect(true).to.deep.equalInAnyOrder(false, 'message'),
```

Comment:
could you rename `message` into `message1` in this and the previous test in order to show it's the message defined here?

`message` is too generic so it could be caused by another mechanism

Reply:
Done.